### PR TITLE
Documentation for Calibration Approaches

### LINKED ISSUE #15544: Document calibration approaches
Document them, ensure they are better shared within the dev team.

Include what the source data is, limitations and applicability as well as a worked example

### PR DESCRIPTION
**Pull request by NickDraper**
This has gathered documentation around the different calibration approaches into a calibration subsection of the concepts pages of the Mantid help.

There are some wiki pages that will need to be deleted and redirected once this is merged.

http://www.mantidproject.org/Tube_Calibration_Examples
http://www.mantidproject.org/Tube_Calibration_Introduction
http://www.mantidproject.org/Python_Code_For_Calibration
http://www.mantidproject.org/After_Tube_Calibration
http://www.mantidproject.org/Visualization_of_Calibrated_Tubes
http://www.mantidproject.org/How_to_reset_detectors_calibration
http://www.mantidproject.org/TubeSpec
http://www.mantidproject.org/TubeCalibFitParams
http://www.mantidproject.org/Tube_calib
http://www.mantidproject.org/CalFile

**To test:**

Look at the help pages starting with Calibration in the Concepts pages.

Fixes #15544.

RELEASE NOTES have been added

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [x] Is the code of an acceptable quality?
- [ ] Does the code conform to the coding standards? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [x] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by FedeMPouzols** [comment]
I'm checking the engineering calibration which looks good overall.

File: docs/source/concepts/calibration/EngineeringCalibration.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+You will need to run a sample which will create good clean peaks at known reference dSpacing positions.  To get a good calibration you will want good statistics on this calibration data.  Vanadium and Ceria are common samples used for calibration.
+
+.. interface:: Engineering Diffraction
+  :align: right
+  :width: 400
+
+Using the Engineering GUI
+#########################
+
+Mantid rpovides a graphical user interface that  calculate can calibrations and
```
Comment (FedeMPouzols) [inline docs/source/concepts/calibration/EngineeringCalibration.rst]: typo: "rpovides" => ""provides".

File: docs/source/concepts/calibration/EngineeringCalibration.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+visualize them.
+
+It is possible to load an existing calibration (as a CSV file) and to
+generate a new calibration file (which becomes the new current
+calibration).
+
+A description of the inte3rface and all of it's controls can be found
```
Comment (FedeMPouzols) [inline docs/source/concepts/calibration/EngineeringCalibration.rst]: typo: "inte3rface"

File: docs/source/concepts/calibration/EngineeringCalibration.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Calibrating individual banks
+############################
+
+:ref:`EnggCalibrate <algm-EnggCalibrate>` allows calibration of indiviual bank within the instrument, which is usefull if only one bank has moved from a previous calibration, or if the calibration failed for a certain area of the instrument.  
+  
+Under the hood
+##############
+
+All of these approaches use the algorithm :ref:`EnggFitPeaks <algm-EnggFitPeaks>`, :ref:`FindPeaks <algm-FindPeaks>` and  :ref:`EnggVanadiumCorrections <algm-EnggVanadiumCorrections>` to find, and fit the recorded peaks and compare them to the expected values.
```
Comment (FedeMPouzols) [inline docs/source/concepts/calibration/EngineeringCalibration.rst]: There's been a recent change in the algorithms and the old `EnggFitPeaks` has been split into `EnggFitPeaks` and `EnggFitDIFCFromPeaks`. So where `EnggFitPeaks` is mentioned `EnggFitDIFCFromPeaks` should also be listed.
This is not important and can be edited later on.

When this is in master I'd update the documentation of the Engg GUI to link to this document about calibration.

File: docs/source/concepts/calibration/EngineeringCalibration.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+:ref:`EnggCalibrate <algm-EnggCalibrate>` allows calibration of indiviual bank within the instrument, which is usefull if only one bank has moved from a previous calibration, or if the calibration failed for a certain area of the instrument.  
+  
+Under the hood
+##############
+
+All of these approaches use the algorithm :ref:`EnggFitPeaks <algm-EnggFitPeaks>`, :ref:`FindPeaks <algm-FindPeaks>` and  :ref:`EnggVanadiumCorrections <algm-EnggVanadiumCorrections>` to find, and fit the recorded peaks and compare them to the expected values.  
+
+The peak functions (shapes):  
+
+* :ref:`BackToBackExponential<func-BackToBackExponential>` (used in EnggFitPeaks, to fit the peaks when accuacy is needed)
```
Comment (FedeMPouzols) [inline docs/source/concepts/calibration/EngineeringCalibration.rst]: typo: "accuacy"

File: docs/source/concepts/calibration/EngineeringCalibration.rst [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+* :ref:`Gaussian<func-Gaussian>` (used in FindPeaks when the a quick validation is needed, but fit accuracy is not vital)
+* :ref:`NeutronBk2BkExpConvPVoigt<func-NeutronBk2BkExpConvPVoigt>` (is the Mantid equivalent to the function usedin GSAS).
```
Comment (FedeMPouzols) [inline docs/source/concepts/calibration/EngineeringCalibration.rst]: There's a few related functions with similar names:
`Bk2BkExpConvPV`, `BackToBackExponentialPV`, `NeutronBk2BkExpConvPVoigt`, see also: #9588.

File: docs/source/concepts/calibration/EngineeringCalibration.rst [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+   This can get very large and cumbersome to deal with for instruments with many detectors.
+*  The :ref:`HDF format<DiffractionCalibrationWorkspace>`.
+   This is that is moe compact and is increasingly being used.
+
+Applying your calibration
+#########################
+
+The result of the calibration (the output table given in OutDetPosTable) is accepted by both :ref:`EnggCalibrate<algm-EnggCalibrate>` and :ref:`EnggFocus<algm-EnggFocus>` which use the columns ‘Detector ID’ and ‘Detector Position’ of the table to correct the detector positions before focussing.  The OutDetPosTable output table can also be used to apply the calibration calculated by this algorithm on any other workspace by using the algorithm :ref:`ApplyCalibration<algm-ApplyCalibration>`.
+
+.. categories:: Calibration
```
Comment (FedeMPouzols) [inline docs/source/concepts/calibration/EngineeringCalibration.rst]: This reads very well and will be very helpul. I wouldn't change it.

**Comment by KarlPalmen** [comment]
I've built the documentation and will check that especially the PSD tube calibration. I thank @FedeMPouzols for paying closer attention to the engineering calibration than I would.

**Comment by KarlPalmen** [comment]
I found ``...\build\docs\html\api\python\techniques\calibration.html`` This appears to deal only with tube calibration, so can't be the correct file.

**Comment by KarlPalmen** [comment]
After a build of the code and then of the documentation, I find it in the concepts page. Ah a careful read of the test instructions tells me that's where to look. But looking at the wrong place has shown documentation that should be removed.

File: docs/source/concepts/Calibration.rst [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+Calibration
+===========
+
+.. contents::
+  :local:
+
+What is Calibration?
+--------------------
+
+Calibration is a process which determines and corrects for differences in the definition of an instrument to reality, generally conducted by peorforming a measurement of a reference compound under known conditions.
```
Comment (KarlPalmen) [inline docs/source/concepts/Calibration.rst]: Wouldn't reference sample be a better description than reference compound?

**Comment by KarlPalmen** [comment]
I find the files that 'should' be removed are linked from a tube calibration file in concepts and so are not really duplicates and should not be removed.

File: docs/source/concepts/calibration/AfterTubeCalibration.rst [hunk 8]
```diff
@@ -0,0 +1,20 @@
+.. _After_Tube_Calibration:
+
+After calibration, on option is to save the workspace to a :ref:`Nexus file <Nexus file>`.
```
Comment (KarlPalmen) [inline docs/source/concepts/calibration/AfterTubeCalibration.rst]: This needs a "there is an option" instead of "on option".

**Comment by KarlPalmen** [comment]
It looks very good, especially for tube calibration. Only a couple of minor changes identified here are needed.

**Comment by KarlPalmen** [comment]
The "on option" in ``after-tube-calibration.rst`` still needs fixing to "there is an option".

**Comment by KarlPalmen** [comment]
Documentation is good, but I cannot find the change in the release notes.

**Comment by KarlPalmen** [comment]
:shipit: despite difficulties finding changes in release notes.

**Comment by FedeMPouzols** [comment]
In "file changed", looking for `release/` will take you to the release notes. There's a release note in every relevant section, and all looks good in general.

**Comment by FedeMPouzols** [comment]
Oh and actually there's a specific commit for release notes in this branch: https://github.com/mantidproject/mantid/pull/16059/commits/d610fe56afa0c55fc931bee6ef677d3d2e5bff14, which I find a very convenient practice and I'd even recommend as a best practice.

**Pull request merged**